checker/src/org/checkerframework/checker/experimental/regex_qual/RegexQualifiedTransfer.java

```diff
@@ -82,6 +82,8 @@ public RegexQualifiedTransfer(QualAnalysis<Regex> analysis) {
                         thenStore.insertValue(firstParam, new RegexVal(0));
                     }
                 } catch (FlowExpressionParseException e) {
+                    // report errors here
+                    // checker.report(e.getResult(), n);
```

Comment:
`analysis.getContext().getCheckerAdapter().report()` should work. But why not just ErrorAbort rather than issue a warning then asserting false?